src/components/snackbar/themes/snackbar.bootstrap.scss

```diff
@@ -0,0 +1,10 @@
+@use '../../../styles/utilities' as *;
+
+[part~='base'] {
+    background: color(surface, 500);
+    color: contrast-color(surface, 500);
+    border: 1px solid color(gray, 300);
+    width: rem(348px);
```

Comment:
I am not sure about this. Maybe it should be max-width, instead of width?

Reply:
The width should be fixed - see [material snackbar](https://material.io/components/snackbars#specs)

Reply:
I don't think so. I see a bunch of examples where the UI is much larger than the set 348 pixels. @StefanIvanov please reconsider.